ibis/pandas/execution/strings.py

```diff
@@ -23,7 +23,7 @@
 
 @execute_node.register(ops.StringLength, pd.Series)
 def execute_string_length_series(op, data, **kwargs):
-    return data.str.len()
+    return data.str.len().astype('int32')
```

Comment:
The output types will be incorrect, therefore otherwise it breaks the tests.
Also `StringLenght` should return `int32` defined in https://github.com/ibis-project/ibis/blob/master/ibis/expr/operations.py#L622